lib/miq_automation_engine/service_models/miq_ae_service_metric_zones.rb

```diff
@@ -0,0 +1,4 @@
+module MiqAeMethodService
+  class MiqAeServiceMetricZones < MiqAeServiceModelBase
```

Comment:
To be clear I believe the `expose` in `miq_ae_service_container_replicator.rb` and this service model should both be removed.